.github/workflows/docs-check.yml

```diff
@@ -27,6 +27,8 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
+         name: run markdownlint
+         run:  make markdownlint
 
```

Comment:
Spacing is still ambiguous. Why isn't yamllint picking this up?

Reply:
@BlaineEXE  wrote:
> Spacing is still ambiguous. Why isn't yamllint picking this up?

I don't see what's ambiguous here.
But the ci reports a syntax error much further up in line 26. 🤷🏼 

https://github.com/rook/rook/actions/runs/13417119834 

And what is the connection with yamllint?

Reply:
Oh I get it now: you are wondering  why yamllint does not fail on this in the linters workflow.

This is because we are explicitly not checking the workflow yamls with `make yamllint` -- only the yamls under `deploy/examples`. maybe we should expand the scope?